netty/build.gradle

```diff
@@ -13,4 +13,4 @@ test {
     jvmArgs "-Xbootclasspath/p:" + configurations.alpnboot.asPath
 }
 
-javadoc.options.links 'https://netty.io/4.1/api/'
+javadoc.options.links 'http://netty.io/4.1/api/'
```

Comment:
This started happening because the https web site went down. I would like to swap back to https if it starts working again.